pkgs/applications/misc/notejot/default.nix

```
@@ -41,6 +41,9 @@ stdenv.mkDerivation rec {
     libgee
   ];
 
+  # FIXME: test if next release fixes this
+  patches = [ ./new-vala.patch];
```

```suggestion
  patches = [ 
    ./new-vala.patch 
  ];
```

Can this be `fetchpatch`ed?

---

Sadly I don't think so, the patch is in the middle of a discussion.